Tools/TestWebKitAPI/Tests/WebKitCocoa/Navigation.mm

```diff
@@ -1714,14 +1702,14 @@ HTTPServer httpServer({
     EXPECT_FALSE(didReceiveAuthenticationChallenge);
     EXPECT_EQ(loadCount, 1);
     EXPECT_WK_STREQ(url, [webView URL].absoluteString);
-
 }
 
 TEST(WKNavigation, HTTPSOnlyInitialLoad)
 {
     using namespace TestWebKitAPI;
     HTTPServer httpsServer({
-        { "/secure"_s, { { }, "secure page"_s } }
+        { "/secure"_s, { { }, "secure page"_s } },
+        { "/notsecure"_s, { { }, "notsecure page upgraded to secure page"_s } }
```

Comment:
Could we rename this path to /initially_not_secure or something to indicate that it is ok that it becomes secure?